src/components/Coupon/index.jsx

```diff
@@ -83,12 +84,13 @@ class Coupon extends React.Component {
 
   renderExpandCollapseIcon() {
     const { isExpanded } = this.state;
-    const iconClass = isExpanded ? 'fa-chevron-up' : 'fa-chevron-down';
+    const icon = isExpanded ? ExpandLess : ExpandMore;
     const iconColor = isExpanded ? 'text-white' : 'text-primary';
     const screenReaderText = isExpanded ? 'Close' : 'Open';
     return (
       <Icon
-        className={classNames('fa', iconClass, iconColor)}
+        className={`align-right ${iconColor}`}
```

Comment:
nit: would recommend keeping the use of `classNames` when working with dynamic class names.